CONTRIBUTING.md

```diff
@@ -53,6 +53,7 @@ This section describes how to start a contribution to Podman Desktop.
 Requirements: 
 * [Node.js 16+](https://nodejs.org/en/)
 * [yarn](https://yarnpkg.com/)
+* [flatpak-builder](https://docs.flatpak.org/en/latest/first-build.html)
```

Comment:
Note that requirement is only required for Linux, so probably we need to add an Optional Linux Requirements section ?